flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/stats/FlinkStatistic.scala

```diff
@@ -86,6 +86,6 @@ object FlinkStatistic {
     * @param tableStats The table statistics.
     * @return The generated FlinkStatistic
     */
-  def of(tableStats: TableStats): FlinkStatistic = new FlinkStatistic(Option(tableStats))
+  def of(tableStats: TableStats): FlinkStatistic = new FlinkStatistic(Some(tableStats))
```

Comment:
`tableStats` may be null, `Option(null)` is `None`. so we should use `Option` instead of `Some` here.